credsweeper/__main__.py

```diff
@@ -270,20 +271,30 @@ def get_json_filenames(json_filename: str):
     return added_json_filename, deleted_json_filename
```

Comment:
It seems there `get_json_filenames()` doesn't seem to be used anywhere..
Could you please remove this function?